index.html

```diff
@@ -50,8 +57,11 @@
             step="20"
           />
         </div>
-        <div id="size-controls-container">
-          <span>Size</span>
+        <div
+          id="size-controls-container"
+          class="container gray highlightTextIn"
+        >
+          <a class="span button" style="font-size: 16px" alt="Size">Size</a>
```

Comment:
Revert it to span and don't change the font size.

Reply:
As we are not using any href attribute, anchor tag works as a span tag itself. Would be useful in future if we add any href attributes. We may not need to edit the css styling when we are adding any href in future. Font Size reverted.

Reply:
I don't think so I have any plans for adding any href here in the near future. So it's better to use the actual tag instead of using some other tag that is meant for some other purposes. Please revert it.

Reply:
Okay, Will do it soon